From 2a0b57dbdfe350d59d2fc85935db3e55c21c47e1 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Sat, 25 May 2024 10:44:17 +0200
Subject: [PATCH] metainfo: Add missing launchable tag

As warned by Flathub's linter:
    "appstream": [
        "E: org.gnome.SoundJuicer.desktop:~: desktop-app-launchable-omitted"
    ],
---
 data/metainfo/org.gnome.SoundJuicer.metainfo.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/metainfo/org.gnome.SoundJuicer.metainfo.xml.in b/data/metainfo/org.gnome.SoundJuicer.metainfo.xml.in
index 04d317a010ad..ea1fe965cd48 100644
--- a/data/metainfo/org.gnome.SoundJuicer.metainfo.xml.in
+++ b/data/metainfo/org.gnome.SoundJuicer.metainfo.xml.in
@@ -2,6 +2,7 @@
 <!-- Copyright 2014 Richard Hughes <richard@hughsie.com> -->
 <component type="desktop">
   <id>org.gnome.SoundJuicer.desktop</id>
+  <launchable type="desktop-id">org.gnome.SoundJuicer.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>Sound Juicer</name>
-- 
2.44.0

